internal/controller/driver_controller.go

```diff
@@ -504,6 +568,9 @@ func (r *driverReconcile) reconcileControllerPluginDeployment() error {
 											utils.CsiAddonsEndpointContainerArg,
 											"",
 										),
+										utils.If(logRotationSpec != nil, utils.LogToStdErrContainerArg, ""),
+										utils.If(logRotationSpec != nil, utils.AlsoLogToStdErrContainerArg, ""),
+										utils.If(logRotationSpec != nil, utils.LogFileContainerArg(fmt.Sprintf("%s-csi-%splugin", r.driver.Name, r.driverType)), ""),
```

Comment:
There is no point in having `csi` in the name, `csi` is already part of the driver name for every and all drivers
```suggestion
										utils.If(logRotationSpec != nil, utils.LogFileContainerArg(fmt.Sprintf("%s-csi-%splugin", r.driver.Name, r.driverType)), ""),
```

Reply:
okay, even I think if we are making changes.
Users upgrading from rook logrotate to csi logrotate will see a change in names, should keep what we have finalised as per rook